.github/workflows/canary-integration-test.yml

```diff
@@ -73,8 +73,19 @@ jobs:
           toolbox=$(kubectl get pod -l app=rook-ceph-tools -n rook-ceph -o jsonpath='{.items[*].metadata.name}')
           timeout 15 sh -c "until kubectl -n rook-ceph exec $toolbox -- ceph mgr dump -f json|jq --raw-output .active_addr|grep -Eosq \"(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\" ; do sleep 1 && echo 'waiting for the manager IP to be available'; done"
           mgr_raw=$(kubectl -n rook-ceph exec $toolbox -- ceph mgr dump -f json|jq --raw-output .active_addr)
+
           timeout 60 sh -c "until kubectl -n rook-ceph exec $toolbox -- curl --silent --show-error ${mgr_raw%%:*}:9283; do echo 'waiting for mgr prometheus exporter to be ready' && sleep 1; done"
 
+      - name: check s5cmd version in the toolbox image
+        run: |
+          toolbox=$(kubectl get pod -l app=rook-ceph-tools -n rook-ceph -o jsonpath='{.items[*].metadata.name}')
+          kubectl -n rook-ceph exec $toolbox -- s5cmd version  | grep -q ^v2.2.1  || {
+            echo " Error: the version of s5cmd version in the  toolbox is not the expected v2.2.1"
+
+             kubectl -n rook-ceph exec $toolbox -- s5cmd version 
+             exit 1
```

Comment:
```suggestion
             exit 1
```
remove this?

Reply:
@subhamkrai  wrote:
> remove this?


not sure, I am not convinced.  I added the exit  because, initially, the ci succeeded despite the version being reported twrongly, so the job succeeded but    printed the error message, which was because after the `||`, there was just the echo command. that's why I added the exit: to ensure overall failure if the version is not correct.

Now the latest failure was because the s5cmd binary was not found in the PATH in the container, and I am trying to fix it differently.

Reply:
@travisn  wrote:

> The s5cmd is only in the operator, and the toolbox pod deployed by the test is the one based on the ceph image. If we want the CI to validate the s5cmd, we would need to launch the toolbox-operator-image.yaml. But I would suggest manual verification is sufficient for this since we don't use s5cmd anywhere except the documentation.

updated to use the toolbox-operator-image pod